bulk-print-fail-alert.tf

```diff
@@ -0,0 +1,18 @@
+module "cmc-bulk-print-fail-alert" {
+  source = "git@github.com:hmcts/cnp-module-metric-alert"
+  location = "${var.location}"
+
+  app_insights_name = "cmc-${var.env}"
+
+  alert_name = "Bulk print failure - CMC"
+  alert_desc = "Triggers when a bulk print failure event is received from CMC in a 5 minute poll."
+  app_insights_query = "customEvents | where name == \"Bulk print failed\""
+  frequency_in_minutes = 5
+  time_window_in_minutes = 5
+  severity_level = "3"
+  action_group_name = "Bulk Print Fail Alert - ${var.env}"
```

Comment:
if you output this from `cnp-module-action-group` then you'll create a dependency which will enforce the order